druid-shell/src/platform/x11/window.rs

```diff
@@ -602,6 +602,42 @@ impl Window {
         }
     }
 
+    pub fn handle_wheel(&self, event: &xcb::ButtonPressEvent) -> Result<(), Error> {
+        let button = event.detail();
+        // The scroll delta of 120.0 was stolen from the GTK code, which apparently came from the
+        // windows docs.
```

Comment:
I think we should rewrite this comment as more to the point. Something like:
*We use a delta of 120 per tick to match the behavior of Windows.*